packages/world/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "prepare": "yarn build && chmod u+x git-install.sh",
     "git:install": "bash git-install.sh",
-    "tablegen": "../cli/dist/mud.js tablegen",
+    "tablegen": "mud tablegen",
```

Comment:
Same here, see comment above (in store/package.json)

Reply:
https://github.com/latticexyz/mud/issues/489